cachi2/core/package_managers/pip.py

```diff
@@ -1837,26 +1891,33 @@ def _add_cachito_hash_to_url(parsed_url: urllib.parse.ParseResult, hash_spec: st
     return parsed_url._replace(fragment=new_fragment).geturl()
 
 
-def _verify_hash(download_path: Path, hashes: list[str]) -> None:
-    """
-    Check that the downloaded archive verifies against at least one of the provided hashes.
+def _to_checksum_info(hash_: str) -> ChecksumInfo:
+    algorithm, _, digest = hash_.partition(":")
+    return ChecksumInfo(algorithm, digest)
 
-    :param download_path: Path to downloaded file
-    :param hashes: All provided hashes for requirement
-    :raise PackageRejected: If computed hash does not match any of the provided hashes
-    """
 
-    def to_checksum_info(hash_: str) -> ChecksumInfo:
-        algorithm, _, digest = hash_.partition(":")
-        return ChecksumInfo(algorithm, digest)
+def _get_user_checksums(
+    req_files: list[PipRequirementsFile],
+) -> dict[tuple[str, str], set[ChecksumInfo]]:
```

Comment:
Is this unused now?

Reply:
I think we should remove this as soon as we realize we won't need it anymore. Maybe add a comment to remind us this is no longer being used?